db/migrate/20151231161821_create_divisions.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDivisions < ActiveRecord::Migration
+  def change
+    create_table :divisions do |t|
+      t.references :organization, index: true, foreign_key: true
```

Comment:
It knows how to create certain ones on databases that support them. Indexes and Foreign Keys are supported. The options on that line creates these lines in `schema.rb`:

`add_index "divisions", ["organization_id"], name: "index_divisions_on_organization_id", using: :btree`

`add_foreign_key "divisions", "organizations"`